theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/Dot.tsx

```diff
@@ -18,9 +18,9 @@ import {useCssCursorLock} from '@theatre/studio/uiComponents/PointerEventsHandle
 import SnapCursor from './SnapCursor.svg'
 import selectedKeyframeIdsIfInSingleTrack from '@theatre/studio/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/selectedKeyframeIdsIfInSingleTrack'
 
-export const dotSize = 6
-const hitZoneSize = 12
-const snapCursorSize = 34
+export const DOT_SIZE = 6
+const HIT_ZONE_SIZE = 12
+const SNAP_CURSOR_SIZE = 34
```

Comment:
Change: Add initial doc comments for what these constants represent and what unit they are in (I assume dp / px?)

Doesn't need to be the best comments ever, just something, since you're probably more aware of what they are than I can be by simply reading the identifier names.

Reply:
I just renamed these using constant styling but I have not touched this code. I looked into a bit and they are in px units. Added _PX postfix.